### PR TITLE
Make the field and point display a flex box

### PR DESCRIPTION
**Pull request by suuruski**
To get the scores shown next to the court

### CODE REVIEW

File: src/badtrainer/ui/core.cljs [hunk 1]
```diff
@@ -98,15 +98,15 @@
  []
  (let [last-points (last (rum/react current-hits))]
   (if (nil? last-points)
```
Comment (sktoiva) [inline src/badtrainer/ui/core.cljs]: Change this to the `if-let` and scrap the `nil?` check.

File: src/badtrainer/ui/core.cljs [hunk 2]
```diff
@@ -98,15 +98,15 @@
...
-  [:div {:style {:float "right"}}
-   [:div "Coordinates: " (str (last-points :coords))]
-   [:div "Shot: " (str (last-points :shot))]
-   [:div "Type: " (str (last-points :type))]
-   [:div "Fault?: " (str (last-points :fault))]
-   [:div "In: " (str (last-points :in))]
-   [:div "Court: " (str (last-points :player-court))]])))
+  [:div {:class :field-text}
+   [:p "Coordinates: " (str (last-points :coords))]
+   [:p "Shot: " (str (last-points :shot))]
```
Comment (sktoiva) [inline src/badtrainer/ui/core.cljs]: I’d keep these as divs. These can be later defined as rows or smthng like that.

**Review by sktoiva — CHANGES_REQUESTED** [review]
Onkos tuo pelkkä `display: flex` riittävä? Kaippa se on.

File: src/badtrainer/ui/core.cljs (hunk 1 above)
Comment (suuruski) [reply src/badtrainer/ui/core.cljs]: Fixed

File: src/badtrainer/ui/core.cljs (hunk 2 above)
Comment (suuruski) [reply src/badtrainer/ui/core.cljs]: Changed

**Comment by suuruski** [comment]
Requests done

**Review by sktoiva — APPROVED** [review]

**Pull request merged**